nucypher/cli/painting.py

```diff
@@ -573,3 +573,31 @@ def paint_locked_tokens_status(emitter, agent, periods) -> None:
         emitter.echo(f"{bucket_range:>9}: {box_plot:60}"
                      f"Min: {NU.from_nunits(bucket_min)} - Max: {NU.from_nunits(bucket_max)}")
 
+
+def paint_input_allocation_file(emitter, allocations) -> None:
+    emitter.echo(f"\n{'='*46} STAGED ALLOCATIONS {'='*45}", bold=True)
+    emitter.echo(f"\n{'Beneficiary':42} | {'Name':20} | {'Duration':20} | {'Amount':20}", bold=True)
+    emitter.echo("-"*(42+3+20+3+20+3+20), bold=True)
+    for allocation in allocations:
+        beneficiary = allocation['beneficiary_address']
+        amount = str(NU.from_nunits(allocation['amount']))
+        duration = (maya.now() + maya.timedelta(seconds=allocation['duration_seconds'])).slang_date()
+        name = allocation.get('name', 'No name provided')
+        emitter.echo(f"{beneficiary} | {name:20} | {duration:20} | {amount:20}")
+    emitter.echo()
+
+
+def paint_deployed_allocations(emitter, allocations, failed) -> None:
+    emitter.echo(f"\n{'='*45} DEPLOYED ALLOCATIONS {'='*44}", bold=True)
+    emitter.echo(f"\n{'Beneficiary':42} | {'Name':20} | {'UserEscrow contract':42} ", bold=True)
+    emitter.echo("-"*(42+3+20+3+42), bold=True)
+    for allocation, contract_address in allocations:
+        beneficiary = allocation['beneficiary_address']
+        name = allocation.get('name', 'No name provided')
+        emitter.echo(f"{beneficiary} | {name:20} | {contract_address}")
+    for allocation in failed:
+        beneficiary = allocation['beneficiary_address']
```

Comment:
How can we get test coverage in this area , :thinking: ? We need a way to mock a transaction failed part of the way through.